pages/kubernetes/how-to/use-sfs-with-kubernetes.mdx

```diff
@@ -0,0 +1,275 @@
+---
+title: How to use SFS with Kubernetes Kapsule
+description: This page explains how to use the Scaleway File Storage Container Storage Interface (CSI) driver to enable Kubernetes users to manage Scaleway File Storage volumes within their clusters.
+tags: kubernetes kubernetes-kapsule kapsule sfs
+dates:
+  validation: 2025-07-18
+  posted: 2025-07-18
+categories:
+  - containers
+  - kubernetes
+---
+import Requirements from '@macros/iam/requirements.mdx'
+
+The Scaleway File Storage Container Storage Interface (CSI) driver enables Kubernetes users to manage Scaleway File Storage volumes within their clusters.  
```

Comment:
> The Scaleway File Storage Container Storage Interface (CSI) driver enables Kubernetes users to manage Scaleway File Storage **volumes** within their clusters.  

Not sure if we want to use the term "File(space here ?)Systems" (File Storage term) or "Volumes" (Kubernetes term) here and in the rest of the doc